src/pageObject.js

```diff
@@ -1,15 +1,42 @@
 'use strict';
 
 class Header {
+  constructor() {
+    this._logo = 'Conduit logo';
+  }
 
+  get logo() {
+    return this._logo;
+  }
 };
 
 class PageObject {
+  constructor(url) {
+    this.url = url;
+    this.header = new Header();
+  }
 
+  clickOnLogo() {
+    return 'Click on the ' + this.header.logo;
+  }
 };
 
 class ArticlePage extends PageObject {
+  super(url, slug) {
```

Comment:
Use 'constructor' instead of 'super' to define the constructor for the ArticlePage class. 'super' is used to call the parent class's constructor within the child class's constructor. 🛠